OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemoryManager.cs

```diff
@@ -36,7 +35,7 @@ private void FindMemory()
             }
             else if (repository.GetMachinaRegion() == GameRegion.Korean)

```

Comment:
Should combine these if statements if the branched code is the same.

```c#
            if (repository.GetMachinaRegion() == GameRegion.Chinese || repository.GetMachinaRegion() == GameRegion.Korean)
            {
                candidates.Add(container.Resolve<ICombatantMemory61>());
            }
```

Reply:
Combined!